airbyte-integrations/connectors/source-klaviyo/source_klaviyo/manifest.yaml

```diff
@@ -770,6 +770,24 @@ definitions:
                     type: string
                   related:
                     type: string
+          attributions:
+            type: ["null", object]
```

Comment:
Based on the test failure, it seems like this should be an array of objects instead of an object. Can we update that?